plugins/goodix-fp/goodixfp.quirk

```diff
@@ -0,0 +1,9 @@
+# Goodix Fingerprint sensor
+[DeviceInstanceId=USB\VID_27C6&PID_60A2]
+Plugin = goodixfp
+Guid = 1542f422-f06b-5c0c-8211-30c102a65f01
+Flags = self-recovery
+Summary = An open source goodix Fingerprint sensor
+Icon = colorimeter-colorhug
```

Comment:
I don't think this is right :) Maybe `auth-fingerprint`?

Reply:
> > protocol in LVFS upload page
> 
> I can add the protocol now to the LVFS if that helps. Quick question: How is the firmware image signed? Some sort of strong crypto more than CRC32?

firmware image encrypt by ECIES,  and image signature with ECDSA.

Reply:
I looked for it in system icons/devices folder  , only this 'colorimeter-colorhug' is the most suitable. this is no icons for fingerprint devices. :)

Reply:
I think we can assume adwaita icon theme is installed, see https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/tree/master/Adwaita/scalable/devices -- it's really *not* a colorimeter.